packages/components/src/components/navigation-item/navigation-item.ts

```diff
@@ -183,20 +200,52 @@ export default class SdNavigationItem extends SolidElement {
               : ''
           }
           <span part="content-container" class="inline-flex items-center flex-auto">
-            <slot part="content" class='inline'></slot>
+            ${
+              this.separated
+                ? html`<a
+                    class=${cx(
+                      'w-full inline-flex items-center px-4 cursor-pointer relative focus-visible:focus-outline hover:bg-neutral-200 group transition-all min-h-[48px]'
+                    )}
+                    href=${ifDefined(isLink ? this.href : undefined)}
+                    target=${ifDefined(isLink ? this.target : undefined)}
+                    download=${ifDefined(isLink ? this.download : undefined)}
+                  >
+                    <slot part="content" class="inline"></slot>
+                  </a>`
+                : html`<slot part="content" class="inline"></slot>`
+            }
           </span>
           ${
             (this.chevron || slots['children']) && this.vertical
-              ? html`<sd-icon
-                  name="chevron-down"
-                  part="chevron"
-                  library="system"
-                  color="currentColor"
-                  class=${cx(
-                    'h-6 w-6 ml-2 transition-all',
-                    isAccordion ? (this.open ? 'rotate-180' : 'rotate-0') : 'rotate-[270deg]'
-                  )}
-                ></sd-icon>`
+              ? this.separated
+                ? html`<button
+                    type="button"
+                    title="chevron-down-button"
+                    class="sd-interactive sd-interactive--reset"
+                    @click=${this.handleClickSummary}
+                  >
+                    <sd-icon
+                      name="chevron-down"
+                      part="chevron"
+                      library="system"
+                      color="currentColor"
+                      tabindex="-1"
```

Comment:
I am not sure this is needed. Please make sure it is before merging.

Reply:
+1